src/PrefsManager.cpp

```diff
@@ -17,6 +17,10 @@
 
 PrefsManager*	PREFSMAN = nullptr;	// global and accessible from anywhere in our program
 
+// NOTE(sukibaby): This is the buffer size used by RageSoundMixBuffer (abbreviated as RSMB),
```

Comment:
I think using an acronym is confusing an unnecessary:
- For starters, RSMBBuffer essentially says Buffer twice lol
- Also just spelling out RageSoundMixBufferSize is really not that bad and is clear.
- I think we either use AudioBufferSize or RageSoundMixBufferSize everywhere but not both. It's confusing when two different names refer to the same thing. I slightly lean AudioBufferSize since RageSoundMixBufferSize leaks internal implementation (RageSound) which I think is unnecessary context from a user perspective for a preference name.